monai/data/utils.py

```diff
@@ -444,22 +447,28 @@ def pickle_operations(data, key=PICKLE_KEY_SUFFIX, is_encode: bool = True):
     return data
 
 
+def collate_meta_tensor_fn(batch, *, collate_fn_map=None):
+    """collate a sequence of meta tensor into a single batched metatensor"""
```

Comment:
```suggestion
    """
    Collate a sequence of meta tensor into a single batched metatensor. This is called by `collage_meta_tensor` and so should not be used as a collate function directly in dataloaders.
    """
```

I think we should say a little more to make it clear what this function is and to use `collage_meta_tensor` instead.